src/locales/nl.json

```diff
@@ -303,7 +319,9 @@
       "empty-xml-el": "Element {el} cannot be empty",
       "empty-xml-attr": "Attribute {attr} on element {el} cannot be empty",
       "child-handle-required": "CA naam is vereist",
-      "publisher-handle-required": "Publisher naam is vereist"
+      "publisher-handle-required": "Publisher naam is vereist",
+      "non-base64-certificate-xml-el": "Element {el} must contain a correctly Base64 encoded self-signed X.509 BPKI certificate",
+      "invalid-registration-data": "Your registration details could not be parsed correctly"
```

Comment:
Deze zijn niet vertaald?

Reply:
meh.. only applicable if you mess with your XML manually. So, perhaps not needed?

That and I have no idea how to make a sensible translation. I mean:
"Uw registratie details konden niet goed ontleed worden"

It just doesn't work I think.. but feel free to suggest text :D